WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TotalFollowersNewUseCase.kt

```diff
@@ -0,0 +1,62 @@
+package org.wordpress.android.ui.stats.refresh.lists.sections.insights.usecases
+
+import kotlinx.coroutines.CoroutineDispatcher
+import org.wordpress.android.R
+import org.wordpress.android.analytics.AnalyticsTracker
+import org.wordpress.android.fluxc.store.StatsStore.InsightType.FOLLOWER_TOTALS
+import org.wordpress.android.fluxc.store.stats.insights.SummaryStore
+import org.wordpress.android.modules.BG_THREAD
+import org.wordpress.android.modules.UI_THREAD
+import org.wordpress.android.ui.stats.refresh.NavigationTarget.ViewTotalFollowersStats
+import org.wordpress.android.ui.stats.refresh.lists.sections.BaseStatsUseCase.StatelessUseCase
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Title
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.TitleWithMore
+import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.ValueWithChartItem
+import org.wordpress.android.ui.stats.refresh.utils.StatsSiteProvider
+import org.wordpress.android.ui.utils.ListItemInteraction
+import org.wordpress.android.util.analytics.AnalyticsTrackerWrapper
+import javax.inject.Inject
+import javax.inject.Named
+
+class TotalFollowersNewUseCase @Inject constructor(
```

Comment:
If you leave the existing one as FollowersTotalsUseCase then this could be rename as just TotalFollowersUseCase without the New in between.  The existing one can go when that card is removed

Reply:
Done! Great, I got rid of the "new" word. I didn't like that word. 😄  b14da0f65f13250fe77706b4db9fe76c719dceca